docs/legal/credits.json

```
@@ -0,0 +1,143 @@
+            "version": "24.09",
+            "licenses": [
+                {
+                    "type": "OTHER",
+                    "link": "https://www.7-zip.org/license.txt"
+                }
+            ]
+        },
+        {
+            "name": "AlphaFS",
+            "link": "https://github.com/alphaleonis/AlphaFS",
+            "version": "2.1.3",
+            "licenses": [
+                {
+                    "type": "MIT",
+                    "link": ""
+                }
+            ]
+        },
+        {
+            "name": "Checksum",
+            "link": "https://github.com/chocolatey/checksum",
+            "version": "0.3.1",
+            "licenses": [
+                {
+                    "type": "Apache-2.0",
+                    "link": ""
+                }
+            ]
+        },
+        {
+            "name": "log4net",
+            "link": "https://github.com/apache/logging-log4net",
+            "version": "rel/2.0.12",
+            "licenses": [
+                {
+                    "type": "Apache-2.0",
+                    "link": ""
+                }
+            ]
+        },
+        {
+            "name": "Microsoft.Bcl.HashCode",
+            "link": "https://www.nuget.org/packages/Microsoft.Bcl.HashCode/1.1.1",
+            "version": "1.1.1",
+            "licenses": [
+                {
+                    "type": "MIT",
+                    "link": "https://licenses.nuget.org/MIT"
+                }
+            ]
+        },
+        {
+            "name": "Microsoft.Web.Xdt",
+            "link": "https://www.nuget.org/packages/Microsoft.Web.Xdt/3.1.0",
+            "version": "3.1.0",
+            "licenses": [
+                {
+                    "type": "Apache-2.0",
+                    "link": "https://licenses.nuget.org/Apache-2.0"
+                }
+            ]
+        },
+        {
+            "name": "Newtonsoft.Json",
+            "link": "https://github.com/JamesNK/Newtonsoft.Json",
+            "version": "13.0.1",
+            "licenses": [
+                {
+                    "type": "MIT",
+                    "link": ""
+                }
+            ]
+        },
+        {
+            "name": "Chocolatey.NuGet.Client",
+            "originalLink": "https://github.com/NuGet/NuGet.Client",
+            "link": "https://github.com/chocolatey/NuGet.Client",
+            "version": "3.4.3",
+            "licenses": [
+                {
+                    "type": "Apache-2.0",
+                    "link": ""
+                }
+            ]
+        },
+        {
+            "name": "Rhino.Licensing",
+            "originalLink": "https://github.com/ayende/rhino-licensing",
+            "link": "https://github.com/chocolatey/rhino-licensing",
+            "version": "1.4.1",
+            "licenses": [
+                {
+                    "type": "BSD-3-Clause",
+                    "link": ""
+                }
+            ]
```

```suggestion
            "name": "Rhino.Licensing",
            "link": "https://github.com/ayende/rhino-licensing",
            "modifiedLink": "https://github.com/chocolatey/rhino-licensing",
            "version": "1.4.1",
            "licenses": [
                {
                    "type": "BSD-3-Clause",
                    "link": "https://github.com/ayende/rhino-licensing/blob/1fc90c984b0c3012465a73afae0a53492c969eb5/license.txt"
                }
            ]
```

After our meeting yesterday, it makes more sense to flip flop these fields a bit. The `originalLink` becomes `modifiedLink` instead.